GNUmakefile.in

```diff
@@ -19,11 +19,11 @@ RST2HTMLFLAGS = --strict
 VPATH = $(rootdir)
 
 rootdir = @rootdir@
-cfgdir  = c
-docdir  = d
-smandir = m
-srcdir  = s
-testdir = t
+cfgdir  = cfg
+docdir  = doc
+smandir = man
+srcdir  = src
+testdir = tests
```

Comment:
Would'nt it be more consistent to use tst for tests to keep using three lettered names?